### PR TITLE
chore: parametrize dbt test, add test case

### PR DESCRIPTION
**Pull request by talagluck**
This is passing when materializing tables with dbt as tables, but not as views. This adds a test case to support that, and parametrizes the pytest for efficiency.

Currently, the view test will raise an error, and pytest will ignore it.

### CODE REVIEW

File: tests/tests/test_dbt.py [hunk 1]
```diff
@@ -7,22 +7,32 @@
 from dbt.cli.main import dbtRunner, dbtRunnerResult
 
 
+@pytest.mark.parametrize("model_name,run_success,query_result",
+                         [
+                             ("table_materialization", True, 10),
+                             pytest.param("view_materialization", True, 10, marks=pytest.mark.xfail),
```
Comment (tychoish) [inline tests/tests/test_dbt.py:13]: to summarize: this now works for tables but not views?

File: tests/tests/test_dbt.py [hunk 2]
```diff
@@ -7,22 +7,32 @@
...
-    dbt_profiles_directory: str = "../fixtures/dbt_project/" # TODO
+    project_directory: str = "tests/fixtures/dbt_project/"
+    dbt_profiles_directory: str = "tests/fixtures/dbt_project/"
 
     with glaredb_connection.cursor() as curr:
         curr.execute("create table dbt_test (amount int)")
         curr.execute(
             "INSERT INTO dbt_test (amount) VALUES (0), (1), (2), (3), (4), (5), (6), (7), (8), (9)"
         )
+        1 == 1
```
[the remark this answers is not in the document]
Comment (talagluck) [reply tests/tests/test_dbt.py:35]: ah, debugging artifact. I removed one but forgot the other.

File: tests/tests/test_dbt.py [hunk 3]
```diff
@@ -36,11 +46,10 @@ def test_dbt_glaredb(
...
-    #     query_result: list = curr.execute(f"select * from {model_name}").fetchall()
-    #
-    # assert len(query_result) == 10
+    assert res.success is run_success
+
+    with glaredb_connection.cursor() as curr:
+        curr.execute(f"select count(*) from {model_name}")
+        result: list = curr.fetchone()
+
+    assert result == (query_result,)
```
[the remark this answers is not in the document]
Comment (talagluck) [reply tests/tests/test_dbt.py:55]: I _think_ this is pythonic also, but updated to make it clearer.

File: tests/tests/test_dbt.py (hunk 1 above)
Comment (talagluck) [reply tests/tests/test_dbt.py:13]: Yes, exactly.

**Pull request merged**